mix.exs

```diff
@@ -14,13 +16,21 @@ defmodule Wallaby.Mixfile do
      description: "Concurrent feature tests for elixir",
      deps: deps(),
      docs: docs(),
-     test_coverage: [tool: ExCoveralls],
+
+     # Custom testing
+     aliases: ["test.all": ["test", "test.drivers"],
+               "test.drivers": &test_drivers/1],
```

Comment:
I'm OK with it just being `test.all`. Thats similar to the pattern used in `db_connection` as well. Overriding `test` seems like it could have unintended side-effects.